image-mode/microshift/config/Containerfile.bootc-rhel9

```diff
@@ -0,0 +1,34 @@
+FROM registry.redhat.io/rhel9/rhel-bootc:9.4
+
+ARG USHIFT_VER=4.17
+RUN if [ -z "${MIRROR_REPO}" ]; then \
+      dnf config-manager --set-enabled "rhocp-${USHIFT_VER}-for-rhel-9-$(uname -m)-rpms" \
+          --set-enabled "fast-datapath-for-rhel-9-$(uname -m)-rpms"; \
+    else \
+      # This is required to update the gpgcheck for repoID
+      repoID=$(echo "${MIRROR_REPO#*://}" | tr '/:' '_'); \
+      dnf config-manager --add-repo "${MIRROR_REPO}" \
+          --add-repo "https://mirror.openshift.com/pub/openshift-v4/$(uname -m)/dependencies/rpms/${USHIFT_VER}-el9-beta" \
```

Comment:
Does this `https://mirror.openshift.com/pub/openshift-v4/...` repo need to be hardcoded in addition to the user-specified `"${MIRROR_REPO}"`?

Reply:
yes this is dependency repo and it is different from mirror repo.

Reply:
I don't find a good name around but adding the comment why this dependencies repo is required.

Reply:
maybe `unreleased-mirror` or such? I'm worried the use of `-beta` is unexpected in some cases, and is really non obvious from the option name.

Reply:
Used `unreleased-mirror` as environment variable to avoid confusion.